portfolio/src/main/webapp/script.js

```diff
@@ -88,19 +91,20 @@ document.addEventListener('DOMContentLoaded', (event) => {
   form.addEventListener('submit', e => {
     e.preventDefault();
     fetch(scriptURL, { method: 'POST', body: new FormData(form), mode: "no-cors"})
-	  .then(response => respond("Thank you! Your rec was sent my way :)"))
+	  .then(response => respond("Thank you for sending :)"))
 	  .catch(error => respond("Uh oh, error: " + error.message))
+    
   });//the event occurred
 
   /**
  *Response to recommendation form
  */
   function respond(responseText) {
-    document.getElementById("form-response").innerHTML = responseText;
-    document.getElementById("form-response").style.visibility = "visible";
+    document.getElementById('form-response').innerHTML += responseText;
+    document.getElementById('form-response').style.visibility = "visible";
     setTimeout(() => {
-        document.getElementById("form-response").innerHTML = "";
-        document.getElementById("form-response").style.visibility = "hidden";
+        document.getElementById('form-response').innerHTML = "";
```

Comment:
You can use single quotes https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes

Reply:
ah yes! I keep getting that mixed up with css!